python/lsst/ap/pipe/matchApFakes.py

```diff
@@ -156,7 +176,19 @@ def trim(row):
         return fakeCat[fakeCat.apply(trim, axis=1)]
 
```

Comment:
I think adding "@staticmethod" here would make it clearer that this is a standalone internal calculation.

Reply:
Eh, I'm fine with it the way it is. Not sure we use similar decorators much anywhere.

Reply:
Added a leading underscore to their names.